devimint/src/main.rs

```diff
@@ -605,6 +606,194 @@ async fn cli_tests(dev_fed: DevFed) -> Result<()> {
         .unwrap();
     assert_eq!(client_ng_reissue_amt, reissue_amount);
 
+    // OUTGOING: fedimint-cli NG pays LND via CLN gateway
```

Comment:
This test is getting quite long now, with most of the code between client and client-ng duplicated. Is it time to delete the old client test code? :)